### PR TITLE
Add package descriptions for Microsoft.AspNetCore.{App, All}

### PR DESCRIPTION
**Pull request by natemcmaster**
Resolves https://github.com/aspnet/Universe/issues/1321

Replaces the placeholder with real description, including a link to the runtime installers which are required to use these packages.

### CODE REVIEW

**Comment by vivmishra** [comment]
Approved for 2.1.5
    Hold checkin till the branch opens.

File: src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj [hunk 1]
```diff
@@ -5,8 +5,12 @@
     <IncludeSymbols>false</IncludeSymbols>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <PackageTags>aspnetcore</PackageTags>
-    <Description>Microsoft.AspNetCore.All</Description>
     <EnableApiCheck>false</EnableApiCheck>
+    <PackageDescription>Provides a default set of API for building an ASP.NET Core application, and also includes API for third-party integrations with ASP.NET Core.
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj]: Sorry I'm late, but need to change `API` --> `APIs` in both instances here.

File: src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj [hunk 2]
```diff
@@ -5,8 +5,12 @@
...
+
+This package requires the ASP.NET Core runtime. This runtime is installed by the .NET Core SDK,
+or can be acquired separately using installers available at https://aka.ms/dotnet-download.
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj]: Should there be an explicit newline splitting this sentence? Would that render weird on NuGet.org? I suggest having it be one line and let the UI stuff do the wrapping.

File: src/Microsoft.AspNetCore.App/Microsoft.AspNetCore.App.csproj [hunk 3]
```diff
@@ -5,8 +5,12 @@
     <IncludeSymbols>false</IncludeSymbols>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <PackageTags>aspnetcore</PackageTags>
-    <Description>Microsoft.AspNetCore.App</Description>
     <EnableApiCheck>false</EnableApiCheck>
+    <PackageDescription>Provides a default set of API for building an ASP.NET Core application.
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.App/Microsoft.AspNetCore.App.csproj]: See 2 earlier comments and apply to here.

**Pull request merged**